src/java.base/share/classes/sun/nio/ch/FileChannelImpl.java

```diff
@@ -1276,11 +1248,8 @@ private int traceImplWrite(ByteBuffer src, long position) throws IOException {
             start = FileWriteEvent.timestamp();
             bytesWritten = implWrite(src, position);
         } finally {
-            long duration = FileWriteEvent.timestamp() - start;
-            if (FileWriteEvent.shouldCommit(duration)) {
-                long bytes = bytesWritten > 0 ? bytesWritten : 0;
-                FileWriteEvent.commit(start, duration, path, bytes);
-            }
+            long bytes = bytesWritten > 0 ? bytesWritten : 0;
+            FileWriteEvent.offer(start, path, bytes);
```

Comment:
The number of bytes written is >=0, it can't be < 0, so I'm not sure why the existing code checks bytesWritten > 0.